hls4ml/converters/keras/core.py

```diff
@@ -103,7 +103,7 @@ def parse_activation_layer(keras_layer, input_names, input_shapes, data_reader,
 
 @keras_handler('BatchNormalization')
 def parse_batchnorm_layer(keras_layer, input_names, input_shapes, data_reader, config):
-    assert('BatchNormalization' in keras_layer['class_name'])
+    assert('batchnorm' in keras_layer['class_name'].lower())
```

Comment:
I suggest keeping the original condition, and adding the new condition in '`or`'